os-eap7-launch/added/launch/messaging.sh

```diff
@@ -108,8 +108,9 @@ function configure_mq() {
 # will configure the thread pool args using cgroups and the formulae provied by https://github.com/apache/activemq-artemis/blob/master/artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/client/ActiveMQClient.java
 function configure_thread_pool() {
   source /opt/run-java/container-limits
-  if [ -n "$CONTAINER_CORE_LIMIT" ]; then
-    local mtp=$(expr 8 \* $CONTAINER_CORE_LIMIT) # max thread pool size
+  if [ -n "$CONTAINER_CORE_LIMIT" ] || [ -n "$JAVA_CORE_LIMIT" ]; then
+    local core_limit=${JAVA_CORE_LIMIT:-${CONTAINER_CORE_LIMIT}}
```

Comment:
I think we need to account for the case where ``JAVA_CORE_LIMIT`` is greater than ``CONTAINER_CORE_LIMIT``

Reply:
I wonder if the proper solution to something like this is to set a CORE_COUNT variable that can be used by everybody else, so this logic is in a single place (or, I suppose, overwrite ``JAVA_CORE_LIMIT`` with ``min(JAVA_CORE_LIMIT, CONTAINER_CORE_LIMIT)``.